src/stories/4-elements/section/section.stories.tsx

```diff
@@ -100,8 +113,39 @@ Grid.args = {
 	},
 };
 
```

Comment:
rename the v2 / v3 stories according to remove the errant space (`V 2` -> `V2`)
```suggestion
GridV2.storyName = "Grid V2";
```

Reply:
set name for `ContainedV2` and `ContainedV3` too